### DISCUSSION TITLE
Decompression with kafka connect configuration

### DISCUSSION THREAD

**Discussion by suresh-tattikota** [strimzi/strimzi-kafka-operator#9931] [Q&A]
Can we use transforms/connecters to decompress the data written to kafka topic  
may be like transforms.gzipdecompression 

may be same as below
    transforms.HoistField.type: org.apache.kafka.connect.transforms.HoistField$Value
    transforms.HoistField.field: originalValue
    transforms.InsertField.type: org.apache.kafka.connect.transforms.InsertField$Value
    transforms.InsertField.timestamp.field: timestamp
    transforms.InsertField.static.field: message
    transforms.InsertField.static.value: 'Hello World'
    transforms.ReplaceField.type: org.apache.kafka.connect.transforms.ReplaceField$Value
    transforms.ReplaceField.blacklist: originalValue

**Comment by scholzj** [comment]
I assume that if you can do it with Kafka Connect outside of Kubernetes you should be able to do it with Kafka Connect on Kubernetes.
